packages/kbn-optimizer/limits.yml

```diff
@@ -116,7 +116,7 @@ pageLoadAssetSize:
   synthetics: 40958
   telemetry: 51957
   telemetryManagementSection: 38586
-  threatIntelligence: 44299
+  threatIntelligence: 92283
```

Comment:
Thanks for looking into what the bundle increase was caused by @PhilippeOberti. 92kb is still under the max budget we've set per plugin, but it's still concerning to me that we're loading react-query at all before the threatIntelligence app is loaded. The "page load bundle size" is the JS which is loaded regardless of which application is loaded, including the dashboard and home page. If you upload your stats.json file into a tool like http://webpack.github.io/analyse/ then you'll be able to see which modules in the main chunk are loaded react-query, and hopefully by knowing the exact problematic modules you'll be able to stick something behind an `await import(...)` and reduce the page load bundle size of threatIntelligence even further.

Reply:
ok I'm going to look into that now!

Reply:
@spalger ok I was able to include everything in a lazy loaded bundle and at the same time fixed some import issues that were messing things up quite a bit. Instead of a bundle increase we now have a decrease in size 😆 🕺